main.go

```diff
@@ -52,6 +53,26 @@ func (c config) AssertValid(pattern string) {
 	}
 }
 
+var (
+	// Statsd is a globally-shared datadog client.
+	Statsd *dogstatsd.Client
+)
+
+
+func configureStatsd() {
+	var err error
+	if Statsd != nil {
+		return
+	}
+	Statsd, err = dogstatsd.New("127.0.0.1:8125", &dogstatsd.Context{
+	    Namespace: "sv-rollout.",
+	    Tags:      []string{"zone:us-east-1a"}, // Apended to every metric
```

Comment:
hardcoded? if we ship a new `sv-rollout` it will run in chicago and ashburn for example too. what's the intention here?